### PR TITLE
2153 manage users

### PR DESCRIPTION
**Pull request by nigel-lowry**
### Context

https://trello.com/c/9pMYw1nY

Turns out this is only content changes (it looks like it's changing the attribute it operates on but Patrick Allum's said that isn't the case)

### Changes proposed in this pull request

Text changes only

### CODE REVIEW

**Comment by nigel-lowry** [comment]
[Display the Manage Users content](https://trello.com/c/9pMYw1nY/2153-display-the-manage-users-content)

File: app/views/school/users/index.html.erb [hunk 1]
```diff
@@ -11,15 +11,13 @@
...
+      <li>add or edit existing accounts</li>
+      <li>change who can access the Support Portal</li>
     </ul>
 
-    <p class="govuk-body govuk-!-margin-top-6">You can also choose to let up to 3 users:</p>
-    <ul class="govuk-list govuk-list--bullet govuk-!-margin-bottom-6">
-      <li>order devices</li>
-      <li>use a <%= govuk_link_to 'Support Portal', 'https://computacenterprod.service-now.com/dfe' %> to report faults and <%= govuk_link_to 'view passwords needed to replace DfE settings', devices_guidance_subpage_path('preparing-microsoft-windows-laptops-and-tablets', anchor: 'getting-local-admin-and-bios-passwords-to-install-your-own-software-and-settings') %> on Microsoft Windows devices</li>
-    </ul>
+    <p class="govuk-body">Up to 3 people from your school can access the Support Portal.</p>
```
Comment (ralph-hawkins) [inline app/views/school/users/index.html.erb:19]: instead of school use @school.institution_type

File: config/locales/en.yml [hunk 2]
```diff
@@ -232,9 +232,9 @@ en:
       order_devices_now: Order devices now
     invite_school_user:
       title: Invite a new user
-      will_they_order_devices: Will they place orders for devices?
-      will_they_order_devices_hint: No more than 3 users will be allowed to place orders.
-      can_order_devices: Yes, give them access to the TechSource website for ordering and a Support Portal for technical help
+      will_they_order_devices: Do you want to give them access to the Support Portal?
+      will_they_order_devices_hint: Up to 3 people can have access to the Support Portal.
```
Comment (ralph-hawkins) [inline config/locales/en.yml:236]: Can we remove 'have' from 'Up to 3 people can have access to the Support Portal'?

**Review by ralph-hawkins — DISMISSED** [review]
Looks good! A couple of minor things.

**Review by ralph-hawkins — APPROVED** [review]
super 👍🏻

**Pull request merged**